featuretools/entityset/deserialize.py

```diff
@@ -1,13 +1,27 @@
 import json
 import os
+import tarfile
+from pathlib import Path
 
+import boto3
 import pandas as pd
 
 from featuretools.entityset.relationship import Relationship
 from featuretools.entityset.serialize import FORMATS
-from featuretools.utils.gen_utils import check_schema_version
+from featuretools.utils import is_python_2
```

Comment:
`is_python_2` is in the gen_utils file so we can include it with the other gen_utils imports